### ISSUE TITLE
Not a bug: MongoDB 5.0+ requires a CPU with AVX support

### ISSUE THREAD

**Issue by aduzsardi** [mongodb/mongodb-kubernetes-operator#1135]
I've seen some issues posted here regarding the operator with mongo v5.x+ not working , not sure if those are related but i wanted to mention that we were using an older server model (Dell PowerEdge R610) , and with these servers you cannot run mongodb versions >=5.x

MongoDB versions 4.x work just fine.

> WARNING: MongoDB 5.0+ requires a CPU with AVX support, and your current system does not appear to have that!
>   see https://jira.mongodb.org/browse/SERVER-54407
>   see also https://www.mongodb.com/community/forums/t/mongodb-5-0-cpu-intel-g4650-compatibility/116610/2
>   see also https://github.com/docker-library/mongo/issues/485#issuecomment-891991814

**Comment by HungryHowies** [comment]
I also have this issue, it unfortante Im unable to use the newer version of MOngoDb on my Windows Server 2019  Hyper-v server.

**Comment by dan-mckean** [comment]
Hi, I'm Dan and I'm the Product Manager for MongoDB's support of Kubernetes.

I'm doing some work right now to try and identify how the Community Operator is being used. The Community Operator is something I inherited when I started at MongoDB, but it doesn't get as much attention from us as we'd like and we're trying to understand how it's used in order to establish its future. It will help us prioritize future issues and PRs raised by the community 🙂

Here's a super short survey (it's much easier for us to review all the feedback that way!): [https://docs.google.com/forms/d/e/1FAIpQLSfwrwyxBSlUyJ6AmC-eYlgW_3JEdfA48SB2i5--_WpiynMW2w/viewform?usp=sf_link](https://docs.google.com/forms/d/e/1FAIpQLSfwrwyxBSlUyJ6AmC-eYlgW_3JEdfA48SB2i5--_WpiynMW2w/viewform?usp=sf_link)

 If you'd rather email me instead: [dan.mckean@mongodb.com](mailto:dan.mckean@mongodb.com?subject=MongoDB%20Community%20Operator%20feedback)

Thank you in advance!
Dan
